guides/common/modules/proc_installing-flatpak-applications-on-project-hosts.adoc

```diff
@@ -7,6 +7,9 @@ Use the command line to install selected applications from the enabled Flatpak r
 
 .Prerequisites
 * Flatpak is installed on the host.
+* Set up Flatpak on the content host that consumes applications from {ProjectServer}.
+For more information, see {RHELDocsBaseURL}9/html/administering_the_system_using_the_gnome_desktop_environment/assembly_installing-applications-using-flatpak_administering-the-system-using-the-gnome-desktop-environment#setting-up-flatpak_assembly_installing-applications-using-flatpak[Setting up Flatpak].
+* Ensure that Flatpak runtime repositories are available to clients alongside application repositories because Flatpak applications, such as `rhel9/firefox-flatpak`, depend on the runtime for installation.
```

Comment:
```suggestion
* Ensure that Flatpak runtime repositories are available to hosts alongside application repositories because Flatpak applications, such as `rhel9/firefox-flatpak`, depend on the runtime for installation.
```

suggestion (non-blocking): Use host in favor of client to align with UI